src/registrar/tests/test_views.py

```diff
@@ -35,10 +35,9 @@ def test_health_check_endpoint(self):
         self.assertContains(response, "OK", status_code=200)
 
     def test_home_page(self):
-        """Home page should be available without a login."""
+        """Home page should NOT be available without a login."""
```

Comment:
Thank you! Do we need to remove the non-auth'd view for the homepage so that only the dashboard is on the homepage?

Reply:
Done.